entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java

```diff
@@ -167,6 +169,20 @@ public EntitySite(String title,
         getUserRoles(); // populate the user roles
     }
 
+    public EntitySite(Site site, boolean includeGroups, boolean justInfo) {
```

Comment:
Seems like you're introducing a new parameter here, justInfo, but it is never used.